### DISCUSSION TITLE
Privacy key and seed pharse show as (00)

### DISCUSSION THREAD

**Discussion by Esmail0karimi** [horizontalsystems/unstoppable-wallet-android#6626] [Show and tell]
I restored my wallet with backup file after that i find out my privacy key and seed pharse show me as (00) 
And when I want to transfer or swap my funds I have error scalar is not in the interval [1'n-1] 
Please explain what happened and how I can solve this problem ??

![IMG_20230920_085850_391.jpg](https://github.com/horizontalsystems/unstoppable-wallet-android/assets/91189643/1520aa02-ce24-403e-af7f-dc04b7a45197)

**Comment by serrq** [comment]
#6623 Twice
